src/main/java/com/fidesmo/tutorials/hellofidesmo/MainActivity.java

```diff
@@ -100,42 +102,40 @@ protected void onNewIntent(Intent intent) {
 
     @Override
     public void tagDiscovered(Tag tag) {
-        Log.i(TAG, "Card detected on the NFC interface!");
+        setMainMessage("Reading card...");
```

Comment:
@miguelcardo because that is part of the instruction/tutorial and it doesn't seem as simple to tell them that they should write the link to a string we've defined :) I'll revert if you don't agree! :)